src/scss/constants.scss

```diff
@@ -320,6 +318,10 @@ $z-layers: (
   @return 0 0 0 2px $color;
 }
 
+@function box-shadow--input-shadow($color) {
```

Comment:
Not the luckiest pick of name I think, sure it is connected to input, but shadow appearing twice in the name...

Reply:
renamed c974b57